### PR TITLE
Initial docs

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        title: 'Spawner',
+        logo: {
+          alt: 'Spawner Logo',
+          src: 'img/logo.svg',
+        },
+        // items: [
+        //   {
```
Comment (pretentious7) [inline docs/docusaurus.config.js:59]: are these going to be uncommented later (with tutorial links and such?)

**Review by pretentious7 — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```
Comment (rolyatmax) [inline docs/README.md:8]: Thoughts on using `npm` instead of `yarn` - just for consistency across our codebases?

File: docs/docs/concepts.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+a unique subdomain under that cluster’s hostname (e.g. `302d996a-6b3a-433b-a1ad-c0ad47ba1575.spawner.dev`).
+
+Spawner is capable of supporting multiple clusters at once, but servers that run backends (**drones**,
+see below) are only capable of belonging to one cluster at a time.
+
+## Controller and Drone
+
+Functionally, Spawner is divided into two parts: the **controller**, and the **drone**.
+
+The controler is the dispatch center, responsible for accepting external requests for backends and
```
Comment (rolyatmax) [inline docs/docs/concepts.md:26]: typo: controler

File: docs/docs/concepts.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+deciding which drone to run them on. The controller is also responsible for routing traffic to the
+appropriate drone by serving DNS, and vouching for drones when they need to request HTTPS certificates.
+
+The drones are the workers that run the actual backends. The Spawner drone codebase runs on each
```
Comment (rolyatmax) [inline docs/docs/concepts.md:30]: Might make sense to swap the drone definition here with the controller definition above. That way you define drones before you reference them in the controller definition.

File: docs/README.md (hunk 2 above)
Comment (paulgb) [reply docs/README.md:8]: Yeah, I've been using npm to deploy it, this is just a readme created by the template. I can update it.

File: docs/docs/concepts.md (hunk 4 above)
Comment (paulgb) [reply docs/docs/concepts.md:30]: done

File: docs/docs/concepts.md (hunk 3 above)
Comment (paulgb) [reply docs/docs/concepts.md:26]: fixed

File: docs/docusaurus.config.js (hunk 1 above)
Comment (paulgb) [reply docs/docusaurus.config.js:59]: I think so. Usually I don't like checking in commented code, but I'm just trying to move quickly here and this is a good way of documenting the structure until we need it.

**Pull request merged**